modules/run-vault/run-vault

```diff
@@ -159,6 +166,11 @@ api_addr      = "$api_addr"
 EOF
 )
 
+if [ $(echo "$VAULT_VERSION 0.10.0" | tr " " "\n" |sort --version-sort| head -n 1) = 0.10.0 ]; then
```

Comment:
Please extract this conditional into a named method (i.e., `vault_version_at_least`). Also, use `[[ ]]` instead of `[ ]`.

Reply:
Just updated with the conditional in its own function.
I ran the automated tests, but I believe it failed because of a limitation on my AWS account.  Is there a more narrow test that I can run or a way to limit the resources that are created in other regions?

Reply:
Is the issue you can only use certain regions? If so, you can force Terratest to do so using the `TERRATEST_REGION ` environment variable: https://github.com/gruntwork-io/terratest/blob/master/modules/aws/region.go#L16

Reply:
You shouldn't need to tweak the timeout. The tests take ~15m to run. If they are timing out, it likely means something is failing. Deploy one of the examples, SSH to it, and check syslog for errors (e.g., during User Data).

Reply:
Ok, I fixed some subnetting pieces on my side and the tests are passing now.